chain/network/src/time.rs

```diff
@@ -0,0 +1,99 @@
+#![allow(dead_code)]
+use once_cell::sync::Lazy;
+use parking_lot::RwLock;
+use std::sync::Arc;
+
+// TODO: consider wrapping these types to prevent interactions
+// with other time libraries, especially the the direct access
+// to the realtime (i.e. not through the Clock).
+pub type Instant = time::Instant;
+pub type Utc = time::OffsetDateTime;
+pub type Duration = time::Duration;
+
+// Instant doesn't have a deterministic contructor,
+// however since Instant is not convertible to an unix timestamp,
+// we can snapshot Instant::now() at the process startup
```

Comment:
This won't get run at a process startup.

Reply:
true, fixed